### ISSUE TITLE
Modelo XGBoost + Optuna + CV

### ISSUE THREAD

**Issue by Yedpt** [Yedpt/Dog_Breed_Identifier_ML#4]
XGBoost multiclase, optimización hiperparámetros con Optuna, StratifiedKFold cross-validation, métricas (accuracy, precision/recall/F1, matriz confusión), verificar overfitting <5%
